styles/RichText.module.scss

```diff
@@ -0,0 +1,83 @@
+.bold {
+    font-weight: $weight--bold;
+}
+
+.italic {
+    font-style: italic;
+}
+
+.paragraphLight {
+    color: var(--color--grey);
+    @include font-size(s);
+    line-height: 30px;
+}
+
+.paragraphIndigo {
+    color: var(--color--dark-indigo);
+    @include font-size(s);
+    line-height: 30px;
+}
+
+.paragraphDark {
+    color: var(--color--white);
+    @include font-size(s);
+    line-height: 30px;
+}
```

Comment:
Refactor as `.paragraphIndigo, .paragraphDark, ... { ... }` for common rules. Or use a seperate `.paragraph` class instead & double up with `styles.paragraph` & `styles.paragraphIndigo`. Or use a more generic utility class based solution with scss?